python/minpy/__init__.py

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env python
```

Comment:
we can do
mxnet.minpy.autograd and mxnet.minpy.optim
but nn and rnn should be in mxnet so that it can work for both imperative and symbolic